src/s1reader/s1_orbit.py

```diff
@@ -245,50 +267,23 @@ def get_orbit_file_from_list(zip_path: str, orbit_file_list: list[str]) -> str:
         t_orbit_start = datetime.datetime.strptime(t_orbit_start[1:], FMT)
 
         # string '.EOF' from end of end time string
-        t_orbit_end = datetime.datetime.strptime(t_orbit_end[:-4], FMT)
+        t_orbit_stop = datetime.datetime.strptime(t_orbit_end[:-4], FMT)
```

Comment:
Just checking- this name change was to make line 275 more logical (since the comment above there has "stop" instead of "end")?
I'm noticing now that this file alone has 26 instances of `end_*` and 11 instances of `stop_*`, and most other files have a combo of "stop" and "end". Not sure if we want to pick one and change the rest to match, but perhaps that can be another pull request after this.

Reply:
I prefer `stop` rather than `end`, because 1) "stopTime" is used in the S1 XML file, and 2) isce2 variable names, and 3) burst member variable [here](https://github.com/opera-adt/s1-reader/blob/71e2c9e589f9e010b4952a574fcda83806561339/src/s1reader/s1_burst_slc.py#L590).

Replacing "end" with "stop" throughout `s1reader` sounds good to me. And yes, I think it's better to do that in another dedicated PR.